.github/jsonnetfile.json

```diff
@@ -8,7 +8,7 @@
           "subdir": "workflows"
         }
       },
-      "version": "main"
+      "version": "124c4d996f9625478a79f1884465e29ea082d224"
```

Comment:
I think we should pin this to a tag in `grafana/loki-release` rather than a specific sha. I have a tag for each release branch in Loki, ie. [loki-2.8.x](https://github.com/grafana/loki-release/releases/tag/loki-2.8.x), so maybe we create one for `main` as well? Pinning to a tag allows us to change the release code without having to change Loki, which is nice when we want to fix a pipeline but rerun it for the same Loki commit.

Reply:
A tag/release can be used here as well, I just don't know what we want to set it to. Do we have a tag/release that is what we should be using going forward, doesn't look like we have a tag for 3.x yet.